src/docs/user/references/Notification-service.adoc

```diff
@@ -495,4 +495,65 @@ Presets are not individual event types like the ones described above; instead, t
 | `All with issues`
 | Jobs
 | Automatically adds the events JOB_IN_ERROR, JOB_FINISHED_WITH_ISSUES, TASK_IN_ERROR, TASK_FINISHED_WITH_ERRORS to the subscription
-|===
\ No newline at end of file
+|===
+
+[[microsoft_teams]]
+=== Microsoft teams notifications
+
+To receive Microsoft Teams notifications using the Notification Third-Party notification method, some configuration must be applied in the Microsoft and the Notification Service sides. +
```

Comment:
"To receive Microsoft Teams notifications using the Notification Third-Party method, some configuration must be applied on both the Microsoft and Notification Service sides."
if you refer to ProActive Notification Service maybe just say it to be clear

Reply:
OK